doc/Type/List.pod6

```diff
@@ -17,6 +17,33 @@ Arrays to have every value of the list stored in a container.
 C<List> implements C<Positional> and as such provides support for
 L<subscripts|/language/subscripts>.
 
+=head1 Immutability
+
+Lists are immutable objects, i.e., neither the number of elements in a list
+nor the elements themselves can be changed. Thus, it is not possible to use
+operations that change the list structure itself such as L<shift|/routine/shift>,
+L<unshift|/routine/unshift>, L<push|/routine/push>, L<pop|/routine/pop>,
+L<splice|/routine/splice>
+and L<binding|/language/operators#index-entry-Binding_operator>.
+
```

Comment:
Maybe use =for code :skip-test<illustrates error> here

Reply:
If you to this, it will not be highlighted. It's better to say that you're skipping the tests and the reason why.

Reply:
I don't think I'm understand. Do you mean something like `L<C<splice>|/routine/splice>`? This is just a list of links so I'm unsure why it would need to use `=for code :skip-test`.

Reply:
Sorry, this referred to the lines below, not above...